lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -262,10 +262,7 @@ def _callback_render(self, result, request, id, version, auth_required=False):
             self._render_message(request, encoded_message)
 
     def _errback_render(self, failure, id):
-        log.error("Request failed:")
-        log.error(failure)
-        log.error(failure.value)
-        log.error(id)
+        log.error("Request failed. Id: %s, Failure: %s", id, failure, exc_info=1)
```

Comment:
unfortunately twisted failure and logging don't work with `exc_info=True`.

There is [log_support.failure()](https://github.com/lbryio/lbry/blob/master/lbrynet/core/log_support.py#L152) that will do what you want:

```
log_support.failure(failure, log, "Request failed. Id: %s, Failure: %s", id)
```